SnakeProject/a_star_test.py

```diff
@@ -0,0 +1,132 @@
+import queue
+
+

```

Comment:
what is the purpose of this file. If it is not used, please remove it from the MR